libs/features/personalization/personalization.js

```diff
@@ -316,78 +304,6 @@ function parsePlaceholders(placeholders, config, selectedVariantName = '') {
   }
   return config;
 }
-
-const fetchEntitlements = async () => {
```

Comment:
Since xLG doesn't yet support all entitlements, are there prod use-cases that would warrant keeping this?

Reply:
I asked around and was told it wasn't being used

Reply:
I have been asked about this feature recently, adding @suhjainadobe for visibility in case this will have an impact.